README.md

```diff
@@ -9,10 +9,28 @@ Contents:
 
 This script downloads plain text files of Western Romance books publicaly avaible on [Smashworks](https://www.smashwords.com/). This website has been used to create popular Machine Learning datasets like [BookCorpus](https://huggingface.co/datasets/bookcorpus).
 
-The source code located in `cmd/smashwords-downloader`.
+The source code located in `cmd/smashwords-downloader`. 
+It can be built into an executable with the command `go build *.go`.
 
 The `main.go` script takes the following arugments:
 ```
   -data_dir string
         directory that the book files will download to (default "./data")
+  
+  -id integer
+        The cooresponding ID for the smashswords url you want to scrape
+        https://www.smashwords.com/books/category/1105/downloads/0/free would have an ID of 1105 (default is 1245 == western romance)
```

Comment:
Adding support for choosing your own category is a great addition.

However, choosing it via ID means the user would have to navigate around smashwords and know how the category IDs work. We could add a separate flag for category name which would make it easier to use, maybe even add a subcommand/flag to get all the possible categories.

We could scrape smashwords once and store the category name to ID map in a static file in this repo, or we could scrape the category IDs from the smashwords homepage every time the script is run. On the homepage all of the category links look like this so scarping them should be doable: ("Adventure" ID is 892)
```html
<a href="/shelves/home/892/any/any">Adventure</a>
```